### ISSUE TITLE
weight decay

### ISSUE THREAD

**Issue by hongyi-zhang** [leehomyc/mixup_pytorch#1]
Hi Harry,

Thanks for your implementation of our work!

Because mixup has a strong regularization effect, the best weight decay parameter for mixup is usually smaller than normal training. For example, you should see mixup accuracy improves if the weight decay is set to 1e-4, which is what we used in the paper.

We didn't search for the best weight decay though, so other choices may lead to further improvement.

Best,

**Comment by leehomyc** [comment]
Hi Hongyi. Thanks for the heads up! I already updated the code and reduced the weight decay.
